src/FlixterHeader.jsx

```diff
@@ -10,15 +11,28 @@ function FlixterHeader({ likedMovies, watchedMovies }) {
   return (
     <header className="App-header">
       <h1 className="title">🎥 Flixter 🎬</h1>
-      <button className="toggleSidebar" onClick={toggleSidebar}>
-        ≡
-      </button>
-      <Sidebar
-        likedMovies={likedMovies}
-        isOpen={isSidebarOpen}
-        toggleSidebar={toggleSidebar}
-        watchedMovies={watchedMovies}
-      />
+      <div>
```

Comment:
Is the app name Flixter ?

Reply:
Still thinking of a name haha, any ideas?

Reply:
Not at the moment, but since it's in more than one place you can assign it to a const that can be imported everywhere. The value can be "Flixter" now, and you can replace it with the new name when ready by changing only one string in all the codebase :)